english/blackAndwhite/blackAndwhite.md

```diff
@@ -14,6 +14,14 @@ Along the direction of the wind, through the window next to the bed, A tall and
 wearing a leather hood and jeans, smirks seeing Christy tremble in fear.
 
 Christy sprints to the window to check if someone is hiding there and sees no one. Stranger drifts into the eternal dark
-and leaves a puzzle behind for her.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.